BREAKING.md

```diff
@@ -13,13 +13,18 @@ There are a few steps you can take to write a good change note and avoid needing
 ## 0.56 Breaking changes
 - [`MessageType.Save` and code that handled it was removed](#messageType-save-and-code-that-handled-it-was-removed)
 - [Removed `IOdspResolvedUrl.sharingLinkToRedeem`](#Removed-IOdspResolvedUrl.sharingLinkToRedeem)
+- [`wait()` methods removed from map and directory](#wait()-methods-removed-from-map-and-directory)
```

Comment:
```suggestion
- [`wait()` methods removed from map and directory](#wait-methods-removed-from-map-and-directory)
```
did it work for you with the parens in the link?

Reply:
Good catch, fixed three instances of this in latest.